app/models/conference.rb

```diff
@@ -387,25 +387,45 @@ def self.user_distribution
   end
 
   ##
-  # Calculates the overall programm hours
+  # Calculates the overall program minutes
   #
   # ====Returns
-  # * +hash+ -> Fixnum hours
-  def current_program_hours
+  # * +hash+ -> Fixnum minutes
+  def current_program_minutes
     events_grouped = events.group(:event_type_id)
     events_counted = events_grouped.count
-    calculate_program_hours(events_grouped, events_counted)
+    calculate_program_minutes(events_grouped, events_counted)
   end
 
   ##
-  # Calculates the overall programm hours since date
+  # Calculates the overall program hours
   #
   # ====Returns
-  # * +hash+ -> Fixnum hours
-  def new_program_hours(date)
+  # * +Fixnum+ -> Fixnum hours. Example: 1.5 gets rounded to 2. 1.3 gets rounded 1.
+  def current_program_hours
+    minutes_standard = 60.to_f
+    hours = (current_program_minutes / minutes_standard).round
```

Comment:
I assumed using a float value, we can round of to the nearest hour. Because right if the minutes is ```110``` then without the float value the rounding off will result in ```1 hour``` instead of ```2 hours```. 

But yes I am removing the variable assignment. Not really needed, i realized.